includes/Hooks.php

```diff
@@ -14,6 +14,7 @@
 use MediaWiki\Extension\AbuseFilter\Hooks\AbuseFilterShouldFilterActionHook;
 use MediaWiki\Extension\AbuseFilter\Variables\VariableHolder;
 use MediaWiki\Extension\CentralAuth\User\CentralAuthUser;
+use MediaWiki\Hook\BeforePageDisplayHook;
 use MediaWiki\Hook\BlockIpCompleteHook;
```

Comment:
Should use `MediaWiki\Output\Hook\BeforePageDisplayHook` and moved alphabetically